tests/rig/runtime.py

```
@@ -33,6 +34,52 @@
+        args += ["-f", str(extra)]
+    return args
+
+
+def _drop_missing_files(args: list[str]) -> list[str]:
+    """Keep only the ``-f <path>`` pairs still on disk.
+
+    A path compose can't read makes it refuse the whole command, so a file that
+    vanished mid-run would block teardown entirely rather than cost one overlay.
+    """
+    kept: list[str] = []
+    for flag, path in zip(args[::2], args[1::2]):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)